<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>814</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1315</width>
    <height>814</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
    <weight>50</weight>
    <italic>true</italic>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="handleWidth">
    <number>16</number>
   </property>
   <widget class="QPushButton" name="loadBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono SemiLight</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(176, 176, 176);
	color: rgb(255,255,255);
}
QPushButton:hover {
	background-color: rgb(16,53,100);
	color:rgb(254,216,37);;
}
QPushButton:disabled {
    background-color: rgb(214, 214, 210);
	color: rgb(255, 255, 255);
}

</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="font">
     <font>
      <family>Cascadia Mono SemiLight</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(176, 176, 176);
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(16,53,100);
	color:rgb(254,216,37);
}
QPushButton:disabled {
    background-color: rgb(214, 214, 210);
	color: rgb(255, 255, 255);
}

</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detecBtn">
    <property name="font">
     <font>
      <family>Cascadia Mono SemiLight</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(176, 176, 176);
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(16,53,100);
	color:rgb(254,216,37);
}
QPushButton:disabled {
    background-color: rgb(214, 214, 210);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Track</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createBtn">
    <property name="font">
     <font>
      <family>Cascadia Mono SemiLight</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(176, 176, 176);
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(16,53,100);
	color:rgb(254,216,37);
}
QPushButton:disabled {
    background-color: rgb(214, 214, 210);
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono SemiLight</family>
     <pointsize>34</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(176,176,176);
	color:rgb(255,255,255);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
